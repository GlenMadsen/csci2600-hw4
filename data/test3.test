r
65
85
q