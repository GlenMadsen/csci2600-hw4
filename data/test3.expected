First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Patroon Manor to Blitman Residence Commons:
	Walk SouthWest to (Intersection 115)
	Walk South to (Rensselaer Apartment Housing Project RAHP B Site)
	Walk South to (Stacwyck Apartments)
	Walk SouthWest to (East Campus Athletic Village Arena)
	Walk South to (Intersection 127)
	Walk West to (Intersection 128)
	Walk West to (Intersection 126)
	Walk SouthWest to (Burdett Avenue Residence Hall)
	Walk West to (Intersection 109)
	Walk SouthWest to (Intersection 118)
	Walk SouthWest to (Intersection 119)
	Walk NorthWest to (Admissions)
	Walk West to (Intersection 105)
	Walk West to (Intersection 108)
	Walk West to (Intersection 132)
	Walk West to (Intersection 131)
	Walk West to (Intersection 130)
	Walk SouthWest to (Pittsburgh Building)
	Walk West to (Intersection 94)
	Walk SouthWest to (Winslow Building)
	Walk South to (Louis Rubin Memorial Approach)
	Walk West to (Blitman Residence Commons)
Total distance: 2511.445 pixel units.
